src/app/containers/Toggle/useToggle.jsx

```diff
@@ -7,6 +7,7 @@ import { ToggleContext } from '#contexts/ToggleContext';
 const envMapping = {
   local: 'local',
   test: 'test',
+  stage: 'test',
```

Comment:
Since you're pointing stage to use test toggles, the list of toggles in toggles/index is superfluous and should be removed, since someone will think they can be independently changed.

Reply:
Removing

Reply:
I see I wasn't clear in my comment - I meant that this line in `Toggle/useToggle.jsx` should stay and the new stage-specific toggles in `src/app/lib/config/toggles/index.js` & the snapshot should be removed, since we don't want Stage to have bespoke logic - just to mimick Test.

Reply:
Yea this fails on the e2e tests. We should keep it for now. This logic will most likely not be needed after we move to B/G

Reply:
Seems like these are required for the cypress tests, so should be kept as they are.